docs/reference/docker_compose.yaml

```diff
@@ -117,6 +117,51 @@ long: |-
 
     Setting the `COMPOSE_IGNORE_ORPHANS` environment variable to `true` will stop docker compose from detecting orphaned
     containers for the project.
+
+
+    ### Use Dry Run mode to test your command
```

Comment:
Comments above apply here too of course

Reply:
docs content on this page now doesn't match what's on `docs/reference/compose.md`. Can we make the same changes as we did above?